knotx-server/src/main/java/io/knotx/server/KnotxServerVerticle.java

```diff
@@ -49,7 +51,9 @@ public void start(Future<Void> fut) throws IOException, URISyntaxException {
 
     configuration.getDefaultFlow().getEngineRouting().forEach((key, value) -> {
       if (key == HttpMethod.POST) {
-        router.route().method(key).handler(BodyHandler.create());
+        BodyHandler bodyHandler = StringUtils.isNoneEmpty(configuration.getFileUploadDirectory()) ?
```

Comment:
`StringUtils#isNoneEmpty` implies that we're dealing with multiple values. Unless I'm reading this wrong, we should be fine with `StringUtils#isNotEmpty` or maybe `StringUtils#isNotBlank`. Not sure if it's possible to get a non-empty, blank value here.